### ISSUE TITLE
AttributeError: module 'certbot.interfaces' has no attribute 'IAuthenticator'

### ISSUE THREAD

**Issue by heeen** [helgeerbe/certbot-dns-ionos#19]
I found this comment mentioning plugins referencing zope

https://github.com/certbot/certbot/issues/9427#issuecomment-1271019939

**Comment by heeen** [comment]
as a workaround for the time being you can install the version prior to certbot 2.0:

```sh
$ sudo apt install python3 python3-venv libaugeas0
$ sudo python3 -m venv /opt/certbot/
$ sudo /opt/certbot/bin/pip install --upgrade pip
$ sudo /opt/certbot/bin/pip install certbot==1.32.0 certbot-apache==1.32.0 acme==1.32.0 certbot-dns-ionos 
```

**Comment by helgeerbe** [comment]
@heeen Thanks for the hint. I've seen that there is certbot 2.0 released, but hadn't time to test it. I will have a look at it this week.

**Comment by helgeerbe** [comment]
Should be fixed with release 2022.11.24.
